Blogs/js/blog-details.js

```diff
@@ -0,0 +1,162 @@
+let id=location.hash.slice(1);
+
+function renderArticle(article){
```

Comment:
Nice job @unadine, Good variable naming, however, you should use es6 to make your code more modern and more readable.